.github/workflows/labeler.yml

```diff
@@ -11,3 +15,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/labeler@8558fd74291d67161a8a78ce36a881fa63b766a9 # v5
+
+      - name: Notify in Slack in case of failure
+        id: slack-notification
+        if: failure() && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

Comment:
do we care about the `workflow_dispatch` even being reported?
May be someone trying something out and is usually triggered with intention.